flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -3823,11 +3827,10 @@
         <Component ID="SAFETY">
             <UseTemplate Name="ASOBO_SAFETY_Push_Template">
                 <LEFT_SINGLE_CODE>
-                    0 (&gt;L:A32NX_MASTER_WARNING)
-                    1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN_L)
+                    1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN)
```

Comment:
**issue**: There is still code using `A32NX_MASTER_WARNING`, whereas this could now be one variable. If we no longer make the distinction between the left and right button, then why do we still need two variables to manage?

Reply:
Because some error must not canceled by master warning(only master warning).

In FWC file, if plane on overspeed or ldg not down situation, master warning should not canceled.

I'm not good at programming but... I can try it.
I'll try to reduce simvar!

Reply:
It should managed by that L:PUSH_AUTOPILOT_MASTERAWARN because some warning should off when press master warning like altitude warning.

I think it will removed when new rust FWC system.

Reply:
Ok. That makes sense. Do note the usages of `A32NX_MASTER_WARNING` on line `1748` and `2852` of `A320_Neo_UpperECAM.js`. I don't know if you intended to keep those?

Reply:
you're right. I test with new code and try to keep that master warning